examples/echo/echo_json.cpp

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I don't see that used. I guess this is the beginning of a utility to help verifying also return results from invocations/queries?   If so, this raises a number of questions:
- couldn't (and shouldn't!) this be generic and hence also should be rather in fabric/bin or alike?
- wouldn't that have been easier scripted in python or alike? (i'd venture, extracting the return result from the peer json can be wrritten in a one-liner python or perl command-line invocation?)
- if we don't complete it, maybe better to leave out of this PR and add to one which does it end-to-end? (With right now just creating a corresponding issue?)

Reply:
the `echo` (as well as the `auction`) cc is not complete. It however performs some operations, and it works now for showing that we can run 2 different cc in serial. That's why it is in this PR, and I will add an issue once merged.

As Marcus pointed out the json machinery here might not be needed, so they might be removed.

Reply:
oh, i definitely dd not want to pull the echo test from the PR.  My comment purely to the json response processing (in regards of not being echo specific, maybe simpler dealt with a script and better included in this PR only if complete/used (compared to a separate PR which might generalize and also instrument the auction test with checks of return value)

Reply:
the echo test has been cleaned up

Reply:
@g2flyer I think Marcus is waiting for your approval

Reply:
@bvavala well, i was waiting for you to re-request the review after my initial review. Didn't know when you were done with your changes.  But i assume that is now the case?

Reply:
sorry @g2flyer , there was no button to push for that (I might have done that some days ago).
Anyway, yes, and I have also rebased

Reply:
actually, there is a button: just click on the "arrow-circle" to the left of the reviewer asking for changes and then s/he will be notified to re-review (that is what marcus has done)